cloudflare-deploy.sh

```diff
@@ -28,6 +28,6 @@ curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-
 cargo-binstall --no-confirm --locked trunk
 
 # web_sys unstable APIs needed for copy to clipboard functionality
```

Comment:
Add this comment to `config_wasm.toml`. The comment is to prevent accidentally breaking the web app by removing the `web_sys_unstable_apis` flag. The web app still compiles if the flag is removed, but copy-to-clipboard will not work.